src/main/java/org/kohsuke/github/GHOrganization.java

```diff
@@ -139,7 +139,7 @@ public GHCreateRepositoryBuilder createRepository(String name) {
      *
      * @see <a href= "https://developer.github.com/v3/teams/#get-team-by-name">documentation</a>
      */
-    public GHTeam getTeam(int teamId) throws IOException {
+    public GHTeam getTeam(long teamId) throws IOException {
```

Comment:
We can create a deprecated version of this method and add new method.  We can't just change it.